.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
-node_js: 10.21.0
```

Comment:
I'm curious how this fixes it! Nice find! Is there a link to a test repo in travis showing it working?

If we need to change node version, we should also do it here: https://github.com/SAP/fundamental-styles/blob/master/.nvmrc

Reply:
I'm still testing it so I'll find out if this works ;)